packages/Webkul/Shop/src/Http/Controllers/Customer/AddressController.php

```diff
@@ -123,13 +107,13 @@ public function update($id, AddressRequest $request)
 
         Event::dispatch('customer.addresses.update.before', $id);
 
-        $customerAddress = $this->customerAddressRepository->update(array_merge($request->all(), [
+        $customerAddress = $this->customerAddressRepository->update(array_merge($request->input(), [
```

Comment:
Pass the exact array example, I have already shown.

Reply:
fixed